core/src/View/Kokkos_ViewCtor.hpp

```diff
@@ -27,6 +27,22 @@ struct SequentialHostInit_t {};
 struct WithoutInitializing_t {};
 struct AllowPadding_t {};
 
+// FIXME: AccessorArg_t needs to be templated
+// This really needs to be templated on the argument type
+// But that adds some more complication, preventing simply
+// copying the code from the above properties.
+// For Sacado we only need a size_t for fad_size
+// We can template this later in a separate change.
+struct AccessorArg_t {
+ private:
+  size_t val{};
+
+ public:
+  KOKKOS_FUNCTION AccessorArg_t() = default;
```

Comment:
```suggestion
  KOKKOS_DEFAULTED_FUNCTION AccessorArg_t() = default;
```

Reply:
Would it be ok to convert this to an aggregate?

Reply:
I started that way and then I got annoyed that I can't write `AccessorArg_t(0)` lol

Reply:
anyway I did change it to an aggregate again.